### ISSUE TITLE
Cannot enable termination notification using Azure CLI

### ISSUE THREAD

**Issue by ecalder6** [Azure/azure-cli#12692]
## Describe the bug
Cannot enable termination notification using Azure CLI. I used the CLI command in this doc: https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-terminate-notification

**Command Name**
`az vmss update`

**Errors:**
```
'NoneType' object has no attribute 'terminate_notification_profile'
Traceback (most recent call last):
Temp\pip-install-9101vebg\knack\knack\cli.py, ln 206, in invoke
azure\cli\core\commands\__init__.py, ln 608, in execute
azure\cli\core\commands\__init__.py, ln 666, in _run_jobs_serially
azure\cli\core\commands\__init__.py, ln 659, in _run_job
Local\Temp\pip-install-9101vebg\six\six.py, ln 703, in reraise
azure\cli\core\commands\__init__.py, ln 636, in _run_job
azure\cli\core\commands\__init__.py, ln 306, in __call__
azure\cli\core\commands\arm.py, ln 566, in handler
azure\cli\core\commands\__init__.py, ln 419, in cached_put
azure\cli\core\commands\__init__.py, ln 414, in _put_operation
azure\cli\command_modules\vm\custom.py, ln 2691, in update_vmss
AttributeError: 'NoneType' object has no attribute 'terminate_notification_profile'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- Deploy a VMSS without enabling terminate notification.
- `az vmss update --resource-group {} --name {} --enable-terminate-notification true --terminate-notification-time 15`

## Expected Behavior
Terminate notification is enabled.

## Environment Summary
```
Windows-10-10.0.17763-SP0
Python 3.6.6

azure-cli 2.2.0

```

**Comment by yungezz** [comment]
hi @arrownj  could you pls have a look? thanks.

**Comment by FrancisGrignon** [comment]
Got the same issue

`Error running command 'az vmss update \
  --name "iis-web-vmss" \
  --resource-group "iis-web-rg" \
  --enable-terminate-notification true \
  --terminate-notification-time 15
': exit status 1. Output: ERROR: The command failed with an unexpected error. Here is the traceback:

ERROR: 'NoneType' object has no attribute 'terminate_notification_profile'
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 608, in execute
    raise ex
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 659, in _run_job
    six.reraise(*sys.exc_info())
  File "/usr/lib/python3.6/site-packages/six.py", line 703, in reraise
    raise value
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 636, in _run_job
    result = cmd_copy(params)
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 306, in __call__
    return self.handler(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/arm.py", line 566, in handler
    result = cached_put(cmd, setter, **setterargs)
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 419, in cached_put
    return _put_operation()
  File "/usr/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 414, in _put_operation
    result = operation(parameters=parameters, **kwargs)
  File "/usr/lib/python3.6/site-packages/azure/cli/command_modules/vm/custom.py", line 2602, in update_vmss
    enable=enable_terminate_notification)
AttributeError: 'NoneType' object has no attribute 'terminate_notification_profile'
WARNING: 
To open an issue, please run: 'az feedback'`

It work fine using the Powershell command:

`Update-AzVmss
  -ResourceGroupName 'iis-web-rg'
  -VMScaleSetName 'iis-web-vmss'
  -TerminateScheduledEvents $true
  -TerminateScheduledEventNotBeforeTimeoutInMinutes 15`

Francis

**Comment by arrownj** [comment]
Hi @ecalder6 @FrancisGrignon , this issue is fixed in CLI 2.4.0. You can upgrade and have a try. Thanks
